Subscriber/ApplePayDirectSubscriber.php

```diff
@@ -170,4 +298,18 @@ private function preventApplePayDirectAsPaymentMethodOnConfirm(Enlight_View $vie
         $view->assign('sUserData', $userData);
     }
 
+    private function isDefaultPaymentMethodApplePay() {
```

Comment:
i would need a unit test for this please, so that its documented and verified for the future

thus please move to a separate class or existing component, and adjust the payment name as argument
then create a few tests to verify the correctly configured apple pay "restrictions" for this use case

thank you

Reply:
Unit test for method added

Reply:
thank you